docs/UserGuide.md

```diff
@@ -408,6 +408,11 @@ company that may come in handy during the internship application process.
 
 </div>
 
+<div markdown="block" class="alert alert-danger">
+:warning: If a prefix is specified, even for optional fields, the parameter cannot be blank. e.g. `p/` only is not valid but `p/94442222` is valid.  
```

Comment:
Small suggestion here to use "value after the prefix" rather than parameter.

Reply:
yup changed. thanks